### ISSUE TITLE
change relation between sales and product

### ISSUE THREAD

**Issue by joao-gabriel** [joao-gabriel/bazar#9]
Sales hasMany Products
Products belongsTo Sales

**Comment by joao-gabriel** [comment]
```
ALTER TABLE sales DROP FOREIGN KEY sales_ibfk_1;
ALTER TABLE `sales` DROP `product_id`;
ALTER TABLE `products` ADD `sale_id` INT NULL DEFAULT NULL ;
```

Issue resolved on commit 26f9cc85db07c0951559cda48a0d35c9dc355a91

**Comment by joao-gabriel** [comment]
```
ALTER TABLE `products` ADD INDEX `fk_sales` (`sale_id`) COMMENT '';

ALTER TABLE `products` ADD FOREIGN KEY (`sale_id`) REFERENCES `sales`(`id`) ON DELETE SET NULL ON UPDATE NO ACTION;
```
